src/lib/easyiot.ts

```diff
@@ -0,0 +1,51 @@
+import * as exposes from './exposes';
+import {Fz, Tz} from './types';
+const ea = exposes.access;
```

Comment:
Since this will likely not be used from other files, please move everything into the `devices/easiot.ts` and remove this file.